recipes-openxt/openxt/openxt-measuredlaunch/seal-system

```diff
@@ -144,8 +144,17 @@ forward)
 
         # PCR4 is first extended with the digest of EV_SEPARATOR
         # See TCG EFI Protocol Specification 5.2 Crypto Agile Log Entry Format
-        ev_separator="9069ca78e7450a285173431b3e52c5c25299e473"
-        [ "${tpm2}" -eq 0 ] && ev_separator="df3f619804a92fdb4057192dc43dd748ea778adc52bc498ce80524c014b81119"
+        if [ "${tpm2}" -ne 0 ]; then
+            # Read the currently used separator from tpm0 sysfs.
+            if [ -e "/sys/kernel/security/tpm0/ascii_bios_measurements" ]; then
+                ev_separator="$(awk '$1 == 4 && $3 == 04 { print $2 }' /sys/kernel/security/tpm0/ascii_bios_measurements)"
+            else
```

Comment:
If awk fails to find a string, `ev_separator` will be empty.  Do we want a fall back for that case, or would things be so broken it's not worth bothering?

Reply:
I do not see any drawback with a fallback if `awk` fails to find the string.